--- cytadela/data/Makefile.am~	2007-06-06 02:11:58.000000000 +0200
+++ cytadela/data/Makefile.am	2007-11-30 00:57:13.000000000 +0100
@@ -7,14 +7,6 @@
 fontsdir  = $(datadir)/cytadela/tex/fonts
 soundsdir = $(datadir)/cytadela/sounds
 
-dist_doc_DATA = \
-doc/Doc_pl.html \
-doc/Cytadela_pl.txt \
-doc/Doc_en.html \
-doc/Cytadela_en.txt \
-doc/Doc_fr.html \
-doc/COPYING
-
 dist_music_DATA = \
 music/Menu.mod \
 music/Intro.mod \
